src/components/ContextBar.test.js

```diff
@@ -0,0 +1,625 @@
+import '@testing-library/jest-dom'
+
+import { fireEvent, render } from '@testing-library/react'
+import { div, h } from 'react-hyperscript-helpers'
+import { MenuTrigger } from 'src/components/PopupTrigger'
+import { Ajax } from 'src/libs/ajax'
+import * as Utils from 'src/libs/utils'
+import { ContextBar } from 'src/pages/workspaces/workspace/analysis/ContextBar'
+import { CloudEnvironmentModal } from 'src/pages/workspaces/workspace/analysis/modals/CloudEnvironmentModal'
+import { tools } from 'src/pages/workspaces/workspace/analysis/notebook-utils'
+import { getGalaxyComputeCost, getGalaxyDiskCost, getPersistentDiskCostHourly, getRuntimeCost, runtimeConfigCost } from 'src/pages/workspaces/workspace/analysis/runtime-utils'
+
+
+const GALAXY_COMPUTE_COST = 10
+const GALAXY_DISK_COST = 1
+const RUNTIME_COST = 0.1
+const PERSISTENT_DISK_COST = 0.01
+
+jest.mock('src/pages/workspaces/workspace/analysis/runtime-utils', () => {
+  const originalModule = jest.requireActual('src/pages/workspaces/workspace/analysis/runtime-utils')
+
+  return {
+    ...originalModule,
+    getGalaxyComputeCost: jest.fn(),
+    getGalaxyDiskCost: jest.fn(),
+    getPersistentDiskCostHourly: jest.fn(),
+    getRuntimeCost: jest.fn(),
+    runtimeConfigCost: jest.fn()
+  }
+})
```

Comment:
A minor stylistic thing, but these can be written a little more concisely as:

```suggestion
jest.mock('src/pages/workspaces/workspace/analysis/runtime-utils', () => ({
  ...jest.requireActual('src/pages/workspaces/workspace/analysis/runtime-utils'),
  getGalaxyComputeCost: jest.fn(),
  getGalaxyDiskCost: jest.fn(),
  getPersistentDiskCostHourly: jest.fn(),
  getRuntimeCost: jest.fn(),
  runtimeConfigCost: jest.fn()
}))
```

Might be nice when there are many modules being mocked.